src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/pdl/dto/PdlPerson.kt

```diff
@@ -89,12 +89,19 @@ data class IdentBolk(
 data class PdlIdent(
     val ident: String,
     val historisk: Boolean,
+    val gruppe: String,
 )
 
 data class PdlIdenter(
     val identer: List<PdlIdent>,
 ) {
     fun gjeldende(): PdlIdent = this.identer.first { !it.historisk }
+
+    fun folkeregisteridenter() = medIdentgrupper(PdlIdentGruppe.FOLKEREGISTERIDENT.name)
+
+    fun aktørIder() = medIdentgrupper(PdlIdentGruppe.AKTORID.name)
+
+    fun medIdentgrupper(vararg identgrupper: String) = copy(identer = this.identer.filter { identgrupper.contains(it.gruppe) })
```

Comment:
For å unngå å bruke `.name` så kunde man ev kunnet haft at denne var av type `PdlIdentGruppe`, så blir bruket litt enklere å skjønne